### PR TITLE
Fixes Manifest Bulbs Showing White

### PR DESCRIPTION
**Pull request by 1Sparky1**
Title.

### CODE REVIEW

**Comment by 1Sparky1** [comment]
!bugfix

**Comment by 1Sparky1** [comment]
!review

File: tgui/packages/tgui/interfaces/common/ManifestSection.tsx [hunk 1]
```diff
@@ -55,7 +55,10 @@ export const ManifestSection = (props, context) => {
                           name="circle"
                           className={
                             'manifest-indicator-' +
-                            crewmate.active.toLowerCase()
+                            crewmate.active
+                              .toLowerCase()
+                              .replace(/\*/g, '')
+                              .replace(/\s/g, '-')
```
Comment (NonQueueingMatt) [inline tgui/packages/tgui/interfaces/common/ManifestSection.tsx:61]: sanitize this in the byond side (i think just doing `ckey()` is enough)

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]

File: tgui/packages/tgui/interfaces/common/ManifestSection.tsx (hunk 1 above)
Comment (NonQueueingMatt) [reply tgui/packages/tgui/interfaces/common/ManifestSection.tsx:61]: actually nevermind you use it later in the ui

**Review by NonQueueingMatt — APPROVED** [review]

**Review by alsoandanswer — APPROVED** [review]

**Pull request merged**